e2e/README.md

```diff
@@ -0,0 +1,13 @@
+# E2E Test for this sample app
```

Comment:
Yeah include the deploy, see: https://github.com/GoogleCloudPlatform/python-compat-runtime/blob/master/tests/e2e/app/Makefile